saleor/graphql/product/mutations/attributes.py

```diff
@@ -0,0 +1,116 @@
+import graphene
+
+from ....product import models
+from ...core.mutations import ModelDeleteMutation, ModelMutation
+
+
+class ProductAttributesInput(graphene.InputObjectType):
+    slug = graphene.String(
+        required=True, description='Internal name.')
+    name = graphene.String(
+        required=True, description='Attribute name.')
```

Comment:
Maybe a better description for `name` would be "Name displayed in the interface".